pkg/bootstrap/config.go

```diff
@@ -283,7 +283,6 @@ func extractRuntimeFlags(cfg *model.NodeMetaProxyConfig) map[string]string {
 		"envoy.reloadable_features.require_strict_1xx_and_204_response_headers":                                "false",
 		"re2.max_program_size.error_level":                                                                     "32768",
 		"envoy.reloadable_features.http_reject_path_with_fragment":                                             "false",
-		"envoy.reloadable_features.no_extension_lookup_by_name":                                                "false",
```

Comment:
This will break EnvoyFilters so probably need to stay on to give enough warning.

Reply:
Good point. Do you know if we can somehow warn based on looking at the actual EnvoyFilter? Alternatively, is there logs in envoy is this flag is enabled but they use extension-by-name?

Reply:
You can parse the final xDS and find all missing typed_configs. I don't think there is a warning in envoy, probably worth adding one.

Reply:
But is there some reasonable way to know something should have a typed_config? I guess some protoreflect magic possibly could do it, any simpler ideas?

Reply:
No, because protobufs don't have subtyping. Yes, this requires some clunky reflection. But also, noone should expect EnvoyFilters to work as-is so this might be solving the wrong problem.

Reply:
Then when can enable this flag?

Reply:
I added a warning in validation